bundles/CoreBundle/Resources/config/routing.yml

```diff
@@ -15,7 +15,7 @@ _pimcore_service_common_files_apple_touch_icon:
         filename: 'apple\-touch\-icon.*'
 
 _pimcore_service_thumbnail:
-    path: '{prefix}image-thumb__{assetId}__{thumbnailName}/{filename}'
+    path: '{prefix}{assetId}/image-thumb__{assetId}__{thumbnailName}/{filename}'
```

Comment:
This is not necessary, right? It's already covered by `{prefix}` and `{assetId}` later in the pattern.

Reply:
Correct, reverted this.